src/components/Input/DropDown/DropDownGroup.js

```diff
@@ -97,7 +106,30 @@ class DropDownGroup extends React.Component {
     return selectedItem && selectedItem.props.children;
   };
 
-  closeDropdown = () => this.setState({ isOpen: false });
+  cleanWord = throttle(() => {
+    this.setState({ word: "", selected: null });
+  }, 1000);
+
+  searchValue = string => {
+    const { children } = this.props;
+
+    const childrenArray = React.Children.toArray(children);
+
+    const firstMatch = childrenArray.find(
+      thisArg =>
+        thisArg.props.children.substring(0, string.length).toLowerCase() ===
```

Comment:
```js
return new Regexp(searchInput).test(option.value);
```
It could help to search as well

Reply:
People usually freak out from regular expressions., but something like this could work:
```return new RegExp(`^${string}`,'i').test(thisArg.props.children);```
I don't know which is more readable.